### ISSUE TITLE
Current NuGet (2.0.0-clinteastwood) is a Debug build on net452

### ISSUE THREAD

**Issue by benaadams** [NancyFx/Nancy#2897]
Could you upload a Release build package?

**Comment by benaadams** [comment]
One on myget looks good (I think)

**Comment by khellang** [comment]
Yeah, I just checked the net452 DLL in https://myget.org/feed/nancyfx/package/nuget/Nancy/2.0.0-pre1909, which is the build of #2898, and it looks to be JIT optimized:

![image](https://user-images.githubusercontent.com/582487/40417817-fd0912be-5e80-11e8-8532-066b49e4db48.png)

**Comment by benaadams** [comment]
https://github.com/TechEmpower/FrameworkBenchmarks/pull/3771 :)

**Comment by khellang** [comment]
Hehe 😝 Are they still taking patches for round 16?
